### DISCUSSION TITLE
Delay on receive messages on Chatwoot self-hosted

### DISCUSSION THREAD

**Discussion by eduardocodes** [chatwoot/chatwoot#10403] [Chatwoot APIs] [answered]
I have Chatwoot self-hosted, which works well, but sometimes the messages I receive get delayed by 1 hour or more, till works well again.

Generally, I try to restart the server, but it doesn't work. Just works randomly after 1 hour I don't know why.

Would I like to know if it is common to happen and if someone has any idea how to solve it? I appreciate the help.

### Server Config:
4 vCPU Cores
6 GB RAM
100 GB NVMe

**Comment by sojan-official** [comment]
please mention the channel type you are using, the chatwoot version as well.

**Reply by eduardocodes** [reply]
I'm using Evolution API (Baileys) to connect Chatwoot

Both on latest versions

**Reply by sojan-official** [reply, marked as answer]
evolution api is not an official Chatwoot supported channel. please reach out to their team as the delays could be based on its implementation
